frontend/src/app/task-area/task-area.component.scss

```diff
@@ -0,0 +1,238 @@
+  border-radius: 8px;
+  display: flex;
+}
+
+.side-bar {
+  padding: 139px 51px;
+  box-sizing: border-box;
+  border-right: 1px solid #D8D8D8;
+  color: #525252;
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  gap: 24px;
+}
+
+.btn {
+  align-self: start;
+  border: none;
+  background: none;
+  font-family: 'Lato';
+  font-style: normal;
+  font-weight: 400;
+  font-size: 22px;
+  line-height: 26px;
+  color:
+    #525252;
+
+}
+
+.active-tab {
+  color: #EA5959;
+}
+
+
+.new-input {
+  font-family: 'Lato';
+  font-style: normal;
+  font-weight: 400;
+  font-size: 18px;
+  line-height: 22px;
+  color: #B5B5B5;
+  padding: 10px 27px 15px;
+  background: #F3F3F3;
+  border-radius: 8px;
+  width: 95%;
+  border: none;
+}
+
+
+.task-block {
+  margin-top: 32px;
+
+  width: 98%;
+
+}
+
+.task-area::-webkit-scrollbar {
+  width: 1px;
+}
+
+.task-area {
+  overflow-y: scroll;
+  height: 100%;
+  box-sizing: border-box;
+  flex-grow: 1;
+  padding: 30px 43px 0px 53px;
+}
+
+.task-list {
+  padding: 0;
+}
+
+.task {
+  margin-bottom: 16px;
+  width: 100%;
+  display: flex;
+  align-items: center;
+  gap: 18px;
+}
+
+
+.custom-checkbox {
+  position: absolute;
+  z-index: -1;
+  opacity: 0;
+}
+
+.custom-checkbox+label {
+  display: inline-flex;
+  align-items: center;
+  user-select: none;
+}
+
+.custom-checkbox+label::before {
+  content: '';
+  display: inline-block;
+  width: 28px;
+  height: 28px;
+  flex-shrink: 0;
+  flex-grow: 0;
+  border: 1px solid #EA5959;
+  box-sizing: border-box;
+  border-radius: 8px;
+  margin-right: 0.5em;
+  background-repeat: no-repeat;
+  background-position: center center;
+  background-size: 50% 50%;
+}
+
+.custom-checkbox:checked+label::before {
+  border-color: #0b76ef;
+  background-color: #0b76ef;
+  background-image: url("data:image/svg+xml,%3csvg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 8 8'%3e%3cpath fill='%23fff' d='M6.564.75l-3.59 3.612-1.538-1.55L0 4.26 2.974 7.25 8 2.193z'/%3e%3c/svg%3e");
+}
+
+.custom-checkbox:not(:disabled):not(:checked)+label:hover::before {
+  border-color: #b3d7ff;
+}
+
+.custom-checkbox:not(:disabled):active+label::before {
+  background-color: #b3d7ff;
+  border-color: #b3d7ff;
+}
+
+.custom-checkbox:focus+label::before {
+  box-shadow: 0 0 0 0.2rem rgba(0, 123, 255, 0.25);
+}
+
+.custom-checkbox:focus:not(:checked)+label::before {
+  border-color: #80bdff;
+}
+
+.custom-checkbox:disabled+label::before {
+  background-color: #e9ecef;
+}
+
+.gaper {
+  flex-grow: 1;
+}
+
+.task-text {
+  font-family: 'Lato';
+  font-style: normal;
+  font-weight: 400;
+  font-size: 18px;
+  line-height: 22px;
+  color: #5A5A5A;
+}
+
+.task-tab {
+  width: 115px;
+  height: 24px;
+  background: #EA5959;
+  border-radius: 4px;
+
+  font-family: 'Lato';
+  font-style: normal;
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 14px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  color: #FFFFFF;
+}
+
+
+
+@media (max-width:1058px) {
```

Comment:
this 1058px and 550px widths - what is the reference to use exactly this values? 
For example Bootstrap has another (as I remember)